### ISSUE TITLE
8. YOLOv8 Model Eğitimi Sorumlusu ve SimCLR sorumlusu

### ISSUE THREAD

**Issue by SenaKymk** [GoktenAI/GoktenVISTA#8]
1. YOLOv8 Yapısını Hazırla
Kullanılacak model: YOLOv8n (hafif test için) veya YOLOv8s (daha güçlü ama yavaş)
data.yaml dosyasını yaz:
yaml
Kopyala
Düzenle
train: ../dataset/train/images
val: ../dataset/val/images
nc: 4
names: ['insan', 'taşıt', 'UAP', 'UAİ']

2. Eğitim Başlat
Hyperparametre önerileri:
batch=8, epochs=100, imgsz=640, lr0=0.01, mosaic=1.0
train.py komut örneği:
bash
Kopyala
Düzenle
yolo task=detect mode=train model=yolov8n.pt data=data.yaml epochs=100 imgsz=640 batch=8

 3. Değerlendirme Metriği Al
runs/detect/train/ içindeki:
results.png (loss, mAP grafikleri)
confusion_matrix.png (sınıflar arası karışma)
Eğitim sonunda .pt modeli models/yolov8n_trained.pt olarak kaydet.

 4. Sınıf Dengesizlik Notu
Eğer bir sınıfta mAP < 0.4 çıkarsa, dengesizlik var demektir. Kişi 1 ile iletişim kurup veri artırma iste. en az 3 kez eğit.
